CONTRIBUTING.md

```diff
@@ -1,63 +1,63 @@
 # Contributing
 
-Thank you so much for contributing to 1Password. We appreciate your time and help. Here are some guidelines to help you get started.
+Thanks for your interest in contributing to the 1Password Connect Terraform Provider project! 🙌 We appreciate your time and effort. Here are some guidelines to help you get started.
 
 ## Building
 
-To build the 1Password Connect Terraform provider run the following:
+Run the following command to build the 1Password Connect Terraform Provider:
 
 ```sh
-$ go build .
+go build .
 ```
 
 This will create the `terraform-provider-onepassword` binary.
 
 ## Testing the Provider
 
-To run the go tests and check test coverage run the following:
+To run the Go tests and check test coverage run the following:
 
 ```sh
-$ go test -v ./... -cover
+go test -v ./... -cover
 ```
 
 ## Generating Documentation
 
-Documentation is generated for the provider using the [terraform documentation plugin](https://github.com/hashicorp/terraform-plugin-docs). This plugin uses the schema `Description` field in conjunction with the contents of the `/templates` and `/examples` folders to generate the `/docs` content.
+Documentation is generated for the provider using [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs). This plugin uses the schema `Description` field in conjunction with the contents of the `/templates` and `/examples` folders to generate the `/docs` content.
 
-To regenerate the `/docs` markdown run:
+To regenerate the `/docs` Markdown run:
 
 ```sh
-$ go generate
+go generate
 ```
 
-## Installing Locally
+## Installing plugin locally
 
-To install the binary it must be copied to the appropriate terraform plugin directory. You may need to create the appropriate directories. For more information [see these Terraform 0.13 plugin docs](https://www.hashicorp.com/blog/automatic-installation-of-third-party-providers-with-terraform-0-13)
+To install the binary it must be copied to the appropriate Terraform plugin directory (note that you may need to create these directories first). You can refer to the [Terraform 0.13 plugin docs](https://www.hashicorp.com/blog/automatic-installation-of-third-party-providers-with-terraform-0-13) for more information.
```

Comment:
Question for reviewers: is referring to Terraform 0.13 required here, or are we just out of date? [v0.13](https://github.com/hashicorp/terraform/releases/tag/v0.13.0) was originally released in August 2020; the latest version is ([v1.5.5](https://github.com/hashicorp/terraform/releases/tag/v1.5.5)) and was released earlier this month.

Reply:
I feel like it's out of date. v0.13 sounds like the version that was available when we first released the provider. And reading the blog post (provided in the link), it seems that it was a new feature which became the standard when Terraform went v1.
With this, I assume the are more extensive docs about this, so might be a good idea to reference to those instead.
And to make this futureproof, we shouldn't mention a version here. We should tell the users to refer to the terraform plugin docs:
- [Prepare Terraform for local provider install](https://developer.hashicorp.com/terraform/tutorials/providers-plugin-framework/providers-plugin-framework-provider#prepare-terraform-for-local-provider-install)
- [Locally install provider and verify with Terraform](https://developer.hashicorp.com/terraform/tutorials/providers-plugin-framework/providers-plugin-framework-provider#locally-install-provider-and-verify-with-terraform)

Reply:
Agree that just purring the links to the Terraform docs should be enough for the users to be able to set it up locally.

Reply:
Removed references to v0.13 and added the Terraform tutorial links that @edif2008 suggested in the [2745e1b](https://github.com/1Password/terraform-provider-onepassword/pull/98/commits/2745e1bb6d8e1f29f4539b7453053d3aba064c52) commit.